src/pygaps/prediction/enthalpy_to_isotherm.py

```diff
@@ -0,0 +1,435 @@
+"""
+A module for predicting isotherms at different pressures given a measured
+isotherm and calculated isosteric heats of adsorption
+"""
+
+import numpy as np
+import pandas as pd
+from scipy import constants
+
+import pygaps.graphing.prediction_graphs
+from pygaps import logger
+from pygaps.characterisation import enthalpy_sorption_whittaker
+from pygaps.core.pointisotherm import PointIsotherm
+from pygaps.utilities.exceptions import ParameterError
+
+R = constants.gas_constant
+
+
+def isotherm_to_predicted(
```

Comment:
This funtion feels like it could be made more generinc. It is currently using the whittaker enthalpy only. If someone has other types of enthalpies of sorption, like from calorimetry, they would be able to make use of this to predict using the clausius clapeyron. Suggestion to make the call to whittaker external to the function